app/onboarding/_components/LoginScreen.tsx

```
@@ -0,0 +1,638 @@
+
+        if (refreshToken) {
+          localStorage.setItem("refresh_token", refreshToken);
+          console.log('✅ Refresh Token 저장됨:', refreshToken);
+          console.log('📍 출처:', refreshTokenFromCookie ? '쿠키' : '헤더');
+        }
+
+        // 응답 본문도 확인
+        const responseData = await response.text();
+        console.log('📄 응답 본문:', responseData);
+
+        return !!(accessToken || refreshToken);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)